contracts/NodeDelegator.sol

```diff
@@ -72,18 +74,53 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
         whenNotPaused
         nonReentrant
         onlySupportedAsset(asset)
-        onlyLRTManager
+        onlyLRTOperator
```

Comment:
nit: The comment needs to be updated as well

Reply:
I'll fix. Thanks